src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -54,6 +67,7 @@ private void Traverse(JsonSchema schema, JToken token, Action<JsonSchema, JValue
                 case JArray array:
                     if (schema.Items != null)
                     {
+                        schema.Items.Parent = schema.Parent;
```

Comment:
still confused about this line

Reply:
go through the json schema, array is different with object